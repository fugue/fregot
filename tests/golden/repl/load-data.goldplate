{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":load data.yaml",
        "data.yaml_policy.filename"
    ]
}
