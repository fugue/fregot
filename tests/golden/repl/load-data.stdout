repl% repl% [36m=[0m [31m"sqlite.c"[0m
repl% 